hack/verify-golint.sh

```diff
@@ -78,16 +90,20 @@ else
 fi
 
 # check to make sure all packages that pass lint are in the linted file.
+echo
 if [ ${#linted[@]} -eq 0 ]; then
 	echo 'Success! All packages that should pass lint are listed in the linted file.'
 else
 	{
-		echo "The following packages passed golint but are not listed in $linted_file:"
+		echo "Some packages passed golint but are not listed in hack/.linted_packages."
+		echo "Please add them in alphabetical order:"
+		echo
 		for p in "${linted[@]}"; do
-			echo "echo $p >> hack/.linted_packages"
+			echo "  echo $p >> hack/.linted_packages"
 		done
+		echo "  LANG=C sort -o hack/.linted_packages hack/.linted_packages"
 		echo
-		echo 'Please add the following packages to the linted file. You can test via this script and commit the result.'
+		echo 'You can test via this script and commit the result.'
```

Comment:
Please open a follow-up PR to fix this. I agree that the wording is not really good. It was even worse before this PR: